### PR TITLE
161108 contributing focus on piv traker not GitHub

### PR DESCRIPTION
**Pull request by weedySeaDragon**
PT Story:  https://www.pivotaltracker.com/story/show/133788131
**CONTRIBUTING.md - revise to emphasize PT, not issues in GitHub**

Changes proposed in this pull request:
1. Changed the text to emphasize Pivotal Tracker as the place where work is developed and discussed, not GitHub.
1.  I took out the diagrams.  They took up a lot of visual space but I don't think they added significantly to the information.
2. I make the first section as short as possible, knowing that is all most people will read.
4. Minor formatting and reorganization to clarify.
   - moved *code style* up so it didn't get buried at the bottom after all of the details

This still may not be _finished_ but I was going too far before getting feedback here.

My questions:
- Did I succeed in describing how this project works?
- Is this clear?
- Would this make sense to someone who was _not_ familiar with AgileVentures or the related bootcamps?
- There are still 'TODO's in this.  Should I leave those in or take them out and create a chore in PT for them?

Ready for review:
@thesuss @lollypop27 @tochman

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -165,20 +216,14 @@ which will mean that the issue is automatically linked to the pull request in th
...
 
 
-#### Pull Request Review
+### Pull Request Review
 
 The project manager(s) will review the pull request for coherence with the specified feature or bug fix, and give feedback on code quality, user experience, documentation and git style.  Please respond to comments from the project managers with explanation, or further commits to your pull request in order to get merged in as quickly as possible.
 
-To maximize flexibility, you can add the project managers as collaborators to your projectscope fork in order to allow them to help you fix your pull request, but this is not required.
-
+To maximize flexibility, you can add the project managers as collaborators to your fork in order to allow them to help you fix your pull request, but this is not required.
```
Comment (diraulo) [inline CONTRIBUTING.md]: Adding managers as collaborators is not really necessary. Github  makes this process simple nowadays. Just ensure when creating a PR That you `Allow edits from maintainers`, and every contributors should be able to push changes to your branch to update the PR.

screenshot 2016-11-12 16 50 26

**Review by diraulo — COMMENTED** [review]
Looks good to me overall. 👍

**Comment by weedySeaDragon** [comment]
Changed the last section:
- allow other members to update your PR,  (changed from old: allow project managers ....)
- clarified last sentences: remove "[WIP]" when you're ready for final review

**Comment by weedySeaDragon** [comment]
@thesuss - what do you think?

**Review by thesuss — APPROVED** [review]

**Pull request merged**